### DISCUSSION TITLE
webp works in Firefox/breaks in Chrome & libvip

### DISCUSSION THREAD

**Discussion by shleeable** [libvips/libvips#3979] [General]
## Bug report
https://social.guckt.info/fileserver/01GZ2GBT34KSYYJ6NJQ9G42FTE/attachment/original/01HV9JAR6HF0823XKYYB2WVAM4.webp

This image seems to fail being opened by libvip and chrome, but it works in firefox.

I applogise for the lack of detail, but this was just found in the wild -- no idea if there is bug or just a corrupt file

**Comment by lovell** [comment]
Looks like a corrupt image, which libwebp (as used by libvips and Chrome) refuses to parse.

```
$ webpinfo -diag 01HV9JAR6HF0823XKYYB2WVAM4.webp 
File: 01HV9JAR6HF0823XKYYB2WVAM4.webp
RIFF HEADER:
  File size: 164828
Error: Truncated data detected when parsing RIFF payload.
Errors detected.
```
